packages/models-library/src/models_library/api_schemas_webserver/projects.py

```diff
@@ -78,14 +78,18 @@ class ProjectGet(OutputSchema):
     ui: EmptyModel | StudyUI | None = None
     quality: dict[str, Any] = {}
     dev: dict | None = None
-    permalink: ProjectPermalink = FieldNotRequired()
-    workspace_id: WorkspaceID | None
-    folder_id: FolderID | None
+    permalink: ProjectPermalink | None = FieldNotRequired()
+    workspace_id: WorkspaceID | None = None
+    folder_id: FolderID | None = None
 
     _empty_description = field_validator("description", mode="before")(
         none_to_empty_str_pre_validator
     )
 
+    model_config = ConfigDict(
+        frozen=False
```

Comment:
Superclass is `frozen` and the behaviour was inherited.

Reply:
thanks! I guess these went off with the re-creation of the branch. how did you find that out? in the warnings of pytest??

Reply:
In a test there was a direct assignment of a field in that model.